src/corelibs/U2Designer/src/PropertyWidget.h

```diff
@@ -249,6 +249,14 @@ protected slots:
 
 private:
     void initModelView();
+
+private:
+    QMap<QString, QString> visibleKeyToBusValueMap;
+    /*
+    * variable to get 'visible value' when it needed instead of writing very similar value() method
+    * also we cant change value() signature because it virtual
+    */
+    bool getVisibleValue;
```

Comment:
Don't use `get` , `set`, `is` or `has` in the beginning of variable names - leave it for getters and setters. Name this variable differently, e.g. `visibleValueHas2BeGot2`, but it's a bit complected too, so, please, turn on your imagination)